src/scripts/generateChangelog.ts

```diff
@@ -16,66 +16,71 @@ async function generateChangelog() {
 
   for (let i = 0; i < args.length; i++) {
     const arg = args[i];
-    if (arg === "--version" || arg === "-v") {
+    if (arg === '--version' || arg === '-v') {
       version = args[i + 1];
-    } else if (arg.startsWith("--version=")) {
-      version = arg.split("=")[1];
-    } else if (!version && !arg.startsWith("-")) {
+    } else if (arg.startsWith('--version=')) {
+      version = arg.split('=')[1];
+    } else if (!version && !arg.startsWith('-')) {
       version = arg;
     }
   }
 
-  const currentTag = getCurrentGitTag()
+  const currentTag = getCurrentGitTag();
   const lastTag = await getLastGitTag();
 
   const config = await loadChangelogConfig(process.cwd(), {
     ...ChangelogenConfig,
     from: currentTag,
-    ...(version && { newVersion: version })
+    ...(version && { newVersion: version }),
   });
 
-  const rawGitCommits = await getGitDiff(
-    currentTag,
-    lastTag
-  );
+  const rawGitCommits = await getGitDiff(currentTag, lastTag);
 
   const newCommits = parseCommits(rawGitCommits, config);
 
   /* Changelog */
   const changelog: string = await generateMarkDown(newCommits, config);
 
   /** Release notes from the changelog without the header */
-  const releaseNotes: string = changelog.split("\n").slice(3).join("\n");
+  const releaseNotes: string = changelog.split('\n').slice(3).join('\n');
 
   process.stdout.write(releaseNotes);
 
   // Update changelog file
   console.info(`Updating ${config.output}`);
-  let changelogMD: string = await Bun.file(config.output, "utf8").text();
+
+  let changelogMD: string;
+  if (typeof config.output === 'string') {
+    // @ts-ignore
+    changelogMD = await Bun.file(config.output, 'utf8').text();
```

Comment:
a) Why can't this be fixed? Try setting `const output = config.output`, then use output instead (getters aren't necessarily stable).
b) Use @ts-expect-error:

```suggestion
    // @ts-expect-error
    changelogMD = await Bun.file(config.output, 'utf8').text();
```

Reply:
Fixed, review